include/seqan3/core/char_operations/all.hpp

```diff
@@ -6,16 +6,21 @@
 // -----------------------------------------------------------------------------------------------------
 
 /*!\file
- * \brief Provides Char predicates for tokenization.
+ * \brief [DEPRECATED] Provides Char predicates for tokenization.
  * \author Rene Rahn <rene.rahn AT fu-berlin.de>
  * \author Hannes Hauswedell <hannes.hauswedell AT fu-berlin.de>
+ * \deprecated This header will be removed in 3.1. Please use seqan3/utility/char_operations/all.hpp instead.
  */
 
 #pragma once
 
 #include <seqan3/utility/char_operations/all.hpp>
 
+SEQAN3_DEPRECATED_HEADER(
+   "This header is deprecated and will be removed in SeqAn-3.1.0; Please #include <seqan3/utility/char_operations/all.hpp> instead.")
+
 /*!\defgroup char_operations builtin_character operations
- * \brief Provides various operations on character types.
+ * \brief [DEPRECATED] Provides various operations on character types.
  * \ingroup core
+ * \deprecated This submodule will be removed in 3.1. Please see the utility/char_operations submodule instead.
```

Comment:
A doxygen group can just be deleted here and newly created in the utility module I think. Otherwise, you may have name conflicts.